### PR TITLE
Fix small gutter size for fluid grid

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
#143

### CODE REVIEW

File: src/app/public/modules/action-button/fixtures/action-button.component.fixture.ts [hunk 1]
```diff
@@ -1,5 +1,10 @@
-import { Component } from '@angular/core';
-import { SkyActionButtonPermalink } from '../types';
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/action-button/fixtures/action-button.component.fixture.ts:2]: ☝️  Not related to this PR, but was throwing a lint error.

File: src/app/public/modules/fluid-grid/fluid-grid.component.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ export class SkyFluidGridComponent {
   }
 
   public get gutterSize(): SkyFluidGridGutterSize {
-    return this._gutterSize || SkyFluidGridGutterSize.Large;
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/fluid-grid/fluid-grid.component.ts:42]: If the consumer initially sets `gutterSize` to `SkyFluidGridGutterSize.Small` this line was evaluating that enum as `0`, and evaluating as `false`, thus reverting to its default of `Large`.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-fix-fluid-grid-small](https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-fix-fluid-grid-small)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-fix-fluid-grid-small](https://host.nxt.blackbaud.com/skyux-layout?_sp.spa=pr-fix-fluid-grid-small)

**Pull request merged**